### ISSUE TITLE
Accuracy on LFW stuck at 75%

### ISSUE THREAD

**Issue by yashbhalgat** [tbmoon/facenet#8]
Hi @tbmoon, thank you for providing this codebase. I have spent almost a month trying to reproduce the accuracy on LFW, but with fail. I am using CASIA to train which is smaller than VGGFace2, but other repos claim equally good accuracy even by training with CASIA. So, there is something I am definitely doing wrong. Can you please let me know what hyperparameters you used to train your model?

Thanks.

**Comment by yashbhalgat** [comment]
Solved. Anyone who faces this issue: use a very small learning rate ~0.0001.
